source/_integrations/kira.markdown

```diff
@@ -129,11 +137,95 @@ repeat:
 
 Some manufacturers (e.g., Samsung) require an IR code to be sent a number of times in a row in rapid succession (usually 3). This doesn't apply to the vast majority of devices, but it can be helpful if needed.
 
+You now need to edit `kira_codes.yaml` to make sure it contains the codes you want the Kira to blast out as a remote or respond to as a sensor. Note that each time you edit and save the `kira_codes.yaml` file you need to restart Home Assistant for the changes to take effect, reloading the automations alone is not sufficient.
+
+### Example remote
+
+Using the number 1 button from a Panasonic DVD player:
+
+```yaml
+# Example kira_codes.yaml entry
+- name: PanaOne
+code: "K 2432 0D31 06EB 0196 01F3 0194 0538 01B4 01D3 01B4 01D1 01B4 01D3 01B4 01D3 01B3 01D3 01B3 01D3 01B4 01D3 01B3 01D3 01B3 01D3 01B4 01D3 01B3 01D3 01B4 0518 01B4 01D3 01B3 01D3 01B4 01D1 01B7 01D1 01B7 01D1 01B4 01D3 01B4 0518 01B4 01D3 01B4 01D1 01B7 0518 01B4 01D3 01B3 01D3 01B4 01D1 01B7 01D1 01B4 01D3 01B4 01D1 01B7 01D1 01B7 01D1 01B4 01D3 01B4 01D1 01B7 01D1 01B4 01D3 01B4 0518 01B4 01D3 01B4 01D1 01B7 01D1 01B7 01D1 01B4 01D3 01B4 01D1 01B7 01D1 01B4 01D3 01B4 01D1 01B7 01D1 01B4 0518 01B7 2000"
+device: DVD
+type: kira
+```
+
+Go to "Configuration" and select "Scripts" and click to add a new script. This is an example using the Panasonic code above.
+
+![alt text](https://raw.githubusercontent.com/Alan-K2/home-assistant.io/patch-2/source/images/integrations/kira_remote_script.png "kira_script")
```

Comment:
Images should be done like this:

```suggestion
<p class='img'>
  <img src='/images/integrations/kira_remote_script.png' />
</p>
```

Adjust all other occurrences as well.

Reply:
At the moment this path gives me a broken link in preview, does the links resolve when this is merged or do I need to submit a separate PR to add images to this repository?

Reply:
The preview does not work fully, because we build the website on publish.
 
You can preview your changes here: <https://deploy-preview-12505--home-assistant-docs.netlify.com/>